src/helpers/discord_utils.ts

```diff
@@ -1698,6 +1718,31 @@ export async function sendInfoWebhook(
     });
 }
 
+/**
+ * Sends an alert to the message webhook
+ * @param webhookURL - The webhook URL
+ * @param embed - the embed payload
+ * @param content - the body text
+ */
+export async function sendInfoEmbedsWebhook(
+    webhookURL: string,
+    embed: EmbedPayload,
+    content: string | undefined,
+): Promise<void> {
+    if (!webhookURL) return;
+    await axios.post(webhookURL, {
+        content,
+        embeds: [
```

Comment:
Does this need a thumbnail URL? Is it missing any default fields send info message has? Can you use the spread operator to fill embed fields instead?

Reply:
Doesn't _need_ it, will just default to the webhook's default avatar/username, but added it as a param anyway. Can't usre spread operator because the field names don't match exactly